ingestion_tools/dataset_configs/template.yaml

```diff
@@ -1,137 +1,187 @@
+    file_format: REQUIRED, STRING
+    filter_value: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    glob_string: REQUIRED, STRING (REGEX)
+    is_visualization_default: OPTIONAL, BOOLEAN (DEFUALT false)
+    mask_label: OPTIONAL, INTEGER (DEFAULT 1) (SemanticSegmentationMask)
+    order: OPTIONAL, STRING (OrientedPoint, InstanceSegmentation)
+    shape: OPTIONAL, ENUM (InstanceSegmentation, OrientedPoint, Point, SegmentationMask, SemanticSegmentationMask)
+dataset_keyphotos: OPTIONAL
+- sources: REQUIRED
+  - literal: REQUIRED
+      value: REQUIRED
+        snapshot: REQUIRED, STRING
+        thumbnail: REQUIRED, STRING
+datasets: REQUIRED
+- metadata: REQUIRED
+    authors: SEE annotations.authors
+    cell_component: OPTIONAL
+      id: OPTIONAL, STRING (GOID)
+      name: REQUIRED, STRING
+    cell_strain:
+      id: OPTIONAL, STRING
+      name: REQUIRED, STRING
+    cell_type: 
+      id: OPTIONAL, STRING
+      name: REQUIRED, STRING
+    cross_references: OPTIONAL
+      dataset_publications: OPTIONAL, STRING (DOIs)
+      related_database_entries: OPTIONAL, STRING (EMPIAR / EMDB IDs)
+    dataset_description: REQUIRED, STRING
+    dataset_identifier: REQUIRED, INTEGER
+    dataset_title: REQUIRED, STRING
+    dates: SEE annotations.dates
+    funding: OPTIONAL BUT RECOMMENDED
+    - funding_agency_name: OPTIONAL BUT RECOMMENDED, STRING
+      grant_id: OPTIONAL BUT RECOMMENDED, STRING
+    grid_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    organism: OPTIONAL
+      name: REQUIRED, STRING
+      taxonomy_id: OPTIONAL, INTEGER
+    sample_preparation: OPTIONAL BUT RECOMMENDED, STRING
+    sample_type: REQUIRED, STRING
+    tissue:
+      id: OPTIONAL, STRING
+      name: REQUIRED, STRING
+  sources: REQUIRED (source_glob, source_multi_glob, destination_glob, literal)
+  # only one finder type per list item, each can have an accompanying parent_filters (SEE runs.sources.parent_filters)
+  - source_glob: OPTIONAL 
+      list_glob: REQUIRED, STRING (REGEX)
+      match_regex: OPTIONAL, STRING (REGEX) (DEFAULT '.*')
+      name_regex: OPTIONAL, STRING (REGEX) (DEFAULT '(.*)')
+  - source_multi_glob:
+      list_globs:
+        - REQUIRED, STRING
+    parent_filters: SEE source_glob.parent_filters
+  - destination_glob: OPTIONAL
+      list_glob: REQUIRED, STRING (REGEX)
+      match_regex: OPTIONAL, STRING (REGEX)
+      name_regex: OPTIONAL, STRING (REGEX)
+    parent_filters: SEE source_glob.parent_filters
+  - literal: OPTIONAL
+      value:
+      - REQUIRED, STRING
+    parent_filters: SEE source_glob.parent_filters
+frames: OPTIONAL
+- sources: SEE datasets.sources and runs.sources.parent_filters (source_glob, source_multi_glob)
+gains: OPTIONAL
+- sources: SEE datasets.sources and runs.sources.parent_filters (source_glob)
+key_images: OPTIONAL
+- sources: SEE dataset.sources (souce_glob, literal)
+rawtilts: OPTIONAL
+- sources: SEE datasets.sources and runs.sources.parent_filters (source_multi_glob)
+runs:
```

Comment:
Hi @manasaV3, could you take a look at this updated parent_filters template structure / documentation?

Reply:
LGTM! 🚀